engine/gamesys/src/gamesys/test/test_gamesys.cpp

```diff
@@ -1432,6 +1435,54 @@ TEST_F(GamepadConnectedTest, TestGamepadConnectedInputEvent)
 
     dmGameSystem::FinalizeScriptLibs(scriptlibcontext);
 }
+
+TEST_F(Sleeping2DCollisionObjectTest, WakingCollisionObjectTest)
+{
+    dmHashEnableReverseHash(true);
+    lua_State* L = dmScript::GetLuaState(m_ScriptContext);
+
+    dmGameSystem::ScriptLibContext scriptlibcontext;
+    scriptlibcontext.m_Factory = m_Factory;
+    scriptlibcontext.m_Register = m_Register;
+    scriptlibcontext.m_LuaState = L;
+    dmGameSystem::InitializeScriptLibs(scriptlibcontext);
+
+    // a 'base' gameobject works as the base for other dynamic objects to stand on
+    const char* path_sleepy_go = "/collision_object/sleepy_base.goc";
+    dmhash_t hash_base_go = dmHashString64("/base-go");
+    // place the base object so that the upper level of base is at Y = 0
+    dmGameObject::HInstance base_go = Spawn(m_Factory, m_Collection, path_sleepy_go, hash_base_go, 0, 0, Point3(50, -10, 0), Quat(0, 0, 0, 1), Vector3(1, 1, 1));
+    ASSERT_NE((void*)0, base_go);
+
+    // two dynamic 'body' object will get spawned and placed appart
```

Comment:
Typos: appart -> apart

Reply:
+1.  i'll wait for jcash also to fix all in one go

Reply:
Fixed.